.identity/env/prod/terraform.tfvars

```diff
@@ -1,8 +1,84 @@
 prefix    = "io"
 env_short = "p"
 env       = "prod"
+location  = "westeurope"
+
+tags = {
+  CreatedBy   = "Terraform"
+  Environment = "Prod"
+  Owner       = "IO"
+  Source      = "https://github.com/pagopa/io-infra"
+  CostCenter  = "TS310 - PAGAMENTI & SERVIZI"
+}
 
 # read-only permissions
+# environment_ci_roles = {
+#   subscription = [
+#     "Reader",
+#     "PagoPA IaC Reader",
+#     "Reader and Data Access",
+#     "Storage Blob Data Reader",
+#     "Storage File Data SMB Share Reader",
+#     "Storage Queue Data Reader",
+#     "Storage Table Data Reader",
+#     "Key Vault Reader",
+#     "DocumentDB Account Contributor",
+#     "API Management Service Contributor",
+#   ]
+# }
+
+github_repository_environment_ci = {
+  protected_branches     = false
+  custom_branch_policies = true
+}
+
+# read-write permissions
+# environment_cd_roles = {
+#   subscription = [
+#     "Contributor",
+#     "Storage Account Contributor",
+#     "Storage Blob Data Contributor",
+#     "Storage File Data SMB Share Contributor",
+#     "Storage Queue Data Contributor",
+#     "Storage Table Data Contributor",
+#     "Key Vault Contributor",
+#   ]
+# }
+
+github_repository_environment_cd = {
+  protected_branches     = true
+  custom_branch_policies = false
+  reviewers_teams = [
+    "infrastructure-admins",
+    "io-backend-admin",
+    "io-backend-contributors",
+  ]
+}
+
+# # --- Managed Identities (new)
+
+ci_github_federations = [
+  {
+    repository = "io-infra"
+    subject    = "prod-ci"
+  },
+  {
+    repository = "io-sign"
```

Comment:
why there's io-sign here?

Reply:
We use a single identity for multiple repositories too keep things simple, so this list contains all the repos to federate

Reply:
I'd rather keep the io-sogn identity apart, ideally all io-sign infra should be moved into a separate subscription oneday. What do you think about it @lucacavallaro ?

Reply:
@lucacavallaro After a chat with @gunzip, we agreed that moving that configuration to `io-sign` repository would be definitely better than this. However, it should be done when other parts of the infrastructure will be moved from `io-infra` to `io-sign` as well. I'm gonna add a comment to remind this